File: src/main/java/org/minturtle/careersupport/CareersupportApplication.java
package org.minturtle.careersupport;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.security.config.annotation.web.reactive.EnableWebFluxSecurity;
import reactor.blockhound.BlockHound;

@SpringBootApplication
@EnableWebFluxSecurity
public class CareersupportApplication {

	public static void main(String[] args) {
		BlockHound.install();
		SpringApplication.run(CareersupportApplication.class, args);
	}

}

File: src/main/kotlin/org/minturtle/careersupport/CareersupportApplication.kt
package org.minturtle.careersupport

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.data.mongodb.repository.config.EnableReactiveMongoRepositories
import org.springframework.security.config.annotation.web.reactive.EnableWebFluxSecurity

@SpringBootApplication
@EnableWebFluxSecurity
@EnableReactiveMongoRepositories
class CareersupportApplication {

    fun main(args: Array<String>) {
        SpringApplication.run(CareersupportApplication::class.java, *args)
    }
}